docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -0,0 +1,404 @@
+                                --load-weights outputs/weights.pth
+                                --save-model-to outputs/nncf
+                                --save-performance outputs/nncf/performance.json
+
+  ...
+
+  2022-12-29 02:11:49,018 | INFO : Loaded model weights from Task Environment
+  2022-12-29 02:11:49,018 | INFO : Model architecture: ATSS
+  2022-12-29 02:11:49,018 | INFO : Loaded model weights from Task Environment
+  2022-12-29 02:11:49,018 | INFO : Model architecture: ATSS
+  2022-12-29 02:11:49,019 | INFO : Task initialization completed
+  INFO:nncf:Please, provide execution parameters for optimal model initialization
+  2022-12-29 02:11:56,996 - mmdet - INFO - Received non-NNCF checkpoint to start training -- initialization of NNCF fields will be done
+  [ INFO ] Received non-NNCF checkpoint to start training -- initialization of NNCF fields will be done
+  [ INFO ]  Received non-NNCF checkpoint to start training -- initialization of NNCF fields will be done
+  2022-12-29 02:11:56,999 - mmdet - INFO - Calculating an original model accuracy
+  ...
+
+  INFO:nncf:Original model accuracy: 0.4319
+  INFO:nncf:Compressed model accuracy: 0.5564
+  INFO:nncf:Absolute accuracy drop: -0.1245
+  INFO:nncf:Relative accuracy drop: -28.82%
+  INFO:nncf:Accuracy budget: 0.1345
+
+
+#TODO significant drop of the loaded snapshot: 0.43 instead of 0.83
+
+#TODO show how to run evaluation of optimized model and its metrics?
+
+#TODO The optimized model isn't being saved (TypeError: cannot pickle '_thread.lock' object)
+
+3. Command example for optimizing OpenVINO model (.xml) with OpenVINO POT:
+
+.. code-block::
+
+  (detection) ...$ otx optimize otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml 
+                                --train-ann-files BBCD/train/_annotations.coco.json 
+                                --train-data-roots  BBCD/train 
+                                --val-ann-files BBCD/valid/_annotations.coco.json 
+                                --val-data-roots BBCD/valid 
+                                --load-weights outputs/openvino/openvino.xml
+                                --save-model-to outputs/pot
+
+  ...
+
+  2022-12-31 05:31:04,125 | INFO : Loading OpenVINO OTXDetectionTask
+  2022-12-31 05:31:05,470 | INFO : OpenVINO task initialization completed
+  2022-12-31 05:31:05,470 | INFO : Start POT optimization
+
+  ...
+
+  2022-12-31 05:37:51,004 | INFO : POT optimization completed
+  2022-12-31 05:37:51,219 | INFO : Start OpenVINO inference
+  2022-12-31 05:37:55,423 | INFO : OpenVINO inference completed
+  2022-12-31 05:37:55,423 | INFO : Start OpenVINO metric evaluation
+  2022-12-31 05:37:55,776 | INFO : OpenVINO metric evaluation completed
+  Performance(score: 0.8343621399176954, dashboard: (1 metric groups))
+
+The POT optimization will take 5-10 minutes without logging.
+
+
+The following stages how to deploy model and run demo are described in [link].
```

Comment:
We already have a dedicated page for that, we may add the link now. The page is empty, but created

Reply:
Fixed